frame/staking/src/pallet/impls.rs

```diff
@@ -983,6 +1008,23 @@ impl<T: Config> Pallet<T> {
 			DispatchClass::Mandatory,
 		);
 	}
+
+	/// Temporary getter for `Payees`.
+	///
+	/// Migrates `Payee` to `Payees` if it has not been migrated already.
+	pub fn get_payout_destination(
```

Comment:
nit, but I found it surprising that a lazy migration was performed when calling a getter. Since this is a temp method, maybe calling it `fn get_payout_destination_migrate` or something along those lines would be more explicit.

Reply:
Amended. I definitely think it is worth having the migration here, as `make_payout` will automatically migrate over the entire active set for an era before `update_payee` or `set_payee` is ever called.